### PR TITLE
fix(service): correctly determine resource age

### PR DESCRIPTION
**Pull request by jsam**

### CODE REVIEW

File: renku/service/cache/models/file.py [hunk 1]
```diff
@@ -52,6 +51,15 @@ def abs_path(self):
         """Full path of cached file."""
         return CACHE_UPLOADS_PATH / self.user_id / self.relative_path
 
+    @property
+    def age(self):
+        """Returns file's age in seconds."""
+        # NOTE: `created_at` field is aligned to UTC timezone.
+        created_at = (self.created_at - datetime.utcfromtimestamp(0)).total_seconds()
+        utc_since = (datetime.utcnow() - datetime.utcfromtimestamp(0)).total_seconds()
+        age = int(utc_since - created_at)
```
Comment (mohammad-sdsc) [inline renku/service/cache/models/file.py:60]: Aren't the three above lines the same as `int((datetime.utcnow() - self.created_at).total_seconds())`?

File: renku/service/cache/models/project.py [hunk 2]
```diff
@@ -52,6 +51,15 @@ def abs_path(self):
         """Full path of cached project."""
         return CACHE_PROJECTS_PATH / self.user_id / self.owner / self.name
 
+    @property
+    def age(self):
+        """Returns project's age in seconds."""
+        # NOTE: `created_at` field is aligned to UTC timezone.
+        created_at = (self.created_at - datetime.utcfromtimestamp(0)).total_seconds()
+        utc_since = (datetime.utcnow() - datetime.utcfromtimestamp(0)).total_seconds()
+        age = int(utc_since - created_at)
```
Comment (mohammad-sdsc) [inline renku/service/cache/models/project.py:60]: Same as above.

**Review by mohammad-sdsc — COMMENTED** [review]
LGTM, just a single comment.

File: renku/service/cache/models/file.py (hunk 1 above)
Comment (jsam) [reply renku/service/cache/models/file.py]: You are absolutely right. Don't know what I was thinking. :eyes:

**Review by mohammad-sdsc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**